Formula/czmqpp.rb

```diff
@@ -0,0 +1,81 @@
+class Czmqpp < Formula
+  desc "C++ wrapper for czmq"
+  homepage "https://github.com/zeromq/czmqpp"
+  url "https://github.com/zeromq/czmqpp.git", :tag => "v1.2.0", :revision => "a3c38eb6e3018887a8ecabc77f3029f7edb7b2d0"
+
+  head "https://github.com/zeromq/czmqpp.git"
+
+  option :universal
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "czmq"
+
+  def install
+    ENV.universal_binary if build.universal?
+
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <iostream>
+      #include <string>
+      #include <vector>
+
+      #include <czmq++/czmqpp.hpp>
+
+      using namespace std;
+
+      int main()
+      {
+        const string addr = "inproc://hello-world";
+        const string msg = "Hello, World!";
+
+        czmqpp::context context;
+
+        czmqpp::socket pull_sock(context, ZMQ_PULL);
+        pull_sock.bind(addr);
+
+        czmqpp::socket push_sock(context, ZMQ_PUSH);
+        push_sock.connect(addr);
+
+        {
+          czmqpp::message send_msg;
+          const czmqpp::data_chunk msg_data(msg.begin(), msg.end());
+          send_msg.append(msg_data);
+          if (!send_msg.send(push_sock))
+            return 1;
+        }
```

Comment:
If I'm interpreting things correctly, you added the braces to limit the scope of the variables in this block. That's a noble goal, but probably overkill in a simple test like this one. I'd suggest to remove them here and in the following block to make the code a bit shorter. To avoid having two `msg_data` you could rename them to `send_data` and `recv_data`.